ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -357,4 +357,10 @@ GTWS:
   opt_uses:
   - thermal_power_capacity
   - differential_pressure_specification
-  - flowrate_requirement
\ No newline at end of file
+  - flowrate_requirement
+  
+HM:
+  description: "Tag for BTU meters"
```

Comment:
heat, not BTU -- thats a unit of measurement.

Reply:
changed to "heat"